seacrowd/sea_datasets/bloom_lm/bloom_lm.py

```diff
@@ -0,0 +1,250 @@
+"""
+SEA Crowd Data Loader for Bloom LM.
+"""
+from typing import Dict, List, Tuple
+
+import datasets
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import TASK_TO_SCHEMA, Licenses, Tasks
+
+_CITATION = r"""
+@inproceedings{leong-etal-2022-bloom,
+    title = "Bloom Library: Multimodal Datasets in 300+ Languages for a Variety of Downstream Tasks",
+    author = "Leong, Colin  and
+      Nemecek, Joshua  and
+      Mansdorfer, Jacob  and
+      Filighera, Anna  and
+      Owodunni, Abraham  and
+      Whitenack, Daniel",
+    editor = "Goldberg, Yoav  and
+      Kozareva, Zornitsa  and
+      Zhang, Yue",
+    booktitle = "Proceedings of the 2022 Conference on Empirical Methods in Natural Language Processing",
+    month = dec,
+    year = "2022",
+    address = "Abu Dhabi, United Arab Emirates",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2022.emnlp-main.590",
+    doi = "10.18653/v1/2022.emnlp-main.590",
+    pages = "8608--8621",
+}
+"""
+
+logger = datasets.logging.get_logger(__name__)
+
+# this config is created for SEACrowd Dataloader
+_LANG_CONFIG = {
+    "abc": "Ambala Ayta",
+    "ahk": "Akha",
+    "bfn": "Bunak",
+    "bjn": "Banjar",
+    "bkx": "Baikeno",
+    "brb": "Brao",
+    "brv": "Western Bru",
+    "bya": "Batak",
+    "bzi": "Bisu",
+    "ceb": "Cebuano",
+    "cgc": "Kagayanen",
+    "cmo": "Central Mnong",
+    "ddg": "Fataluku",
+    "dmg": "Upper Kinabatangan",
+    "dnw": "Western Dani",
+    "dtp": "Kadazan Dusun",
+    "dtr": "Lotud",
+    "enc": "En",
+    "fil": "Filipino",
+    "gal": "Galolen",
+    "hil": "Hiligaynon",
+    "hre": "Hre",
+    "hro": "Haroi",
+    "idt": "Idaté",
+    "ilo": "Ilocano",
+    "ind": "Indonesian",
+    "jra": "Jarai",
+    "kak": "Kalanguya",
+    "khb": "Lü",
+    "khm": "Khmer",
+    "kqr": "Kimaragang",
+    "krr": "Krung",
+    "ksw": "S’gaw Karen",
+    "kvt": "Lahta",
+    "lao": "Lao",
+    "lhu": "Lahu",
+    "llg": "Lole",
+    "lsi": "Lacid",
+    "lwl": "Eastern Lawa",
+    "mdr": "Mandar",
+    "mgm": "Mambae",
+    "mhx": "Lhao Vo",
+    "mkz": "Makasae",
+    "mnw": "Mon",
+    "mqj": "Mamasa",
+    "mry": "Mandaya",
+    "msb": "Masbatenyo",
+    "mya": "Burmese",
+    "nod": "Northern Thai",
+    "nst": "Tangshang Naga",
+    "nxa": "Nauete",
+    "nxl": "South Nuaulu",
+    "pag": "Pangasinan",
+    "pce": "Ruching Palaung",
+    "pdu": "Kayan",
+    "pea": "Peranakan Indonesian",
+    "pmf": "Pamona",
+    "psp": "Filipino Sign Language",
```

Comment:
Current assumption is that `psp` refers to the sign language movement rather than the transcription as used in this dataset. To sort out this confusion, @jcblaisecruz02 will talk to some linguists to see whether this subset is more appropriated to be labeled as `psp` or another written/spoken language (e.g., `ceb`, `tgl`, etc.).

Reply:
As per @jcblaisecruz02's discussion with the linguists, let's mark the `psp` as `psp_ceb`, @sabilmakbar.